[runtime-audit-engine] Add fstec falco rules (#5787)
deckhouse/deckhouse: ee/modules/650-runtime-audit-engine/docs/README.md

> There are several built-in rules that cannot be disabled.  
> These rules are aimed at detecting Deckhouse security problems as well as security problems affecting the `runtime-audit-engine` module.

Reviewer: ```suggestion
There is a built-in set of rules that cannot be disabled. It helps to identify problems with Deckhouse security as well as security problems affecting the `runtime-audit-engine` module.
```
